### PR TITLE
[DIA-2102] Added `scriptVersion` and `scriptType` to  all `Url`s and logger requests

### PR DESCRIPTION
**Pull request by Nevazhnovu**

### CODE REVIEW

**Comment by andresilveirah** [comment]
Looks good to me @Nevazhnovu just make sure to run the tests before opening the PR :D

**Comment by carmelo-iriti** [comment]
@andresilveirah i think that for this request 9 file edited are a lot, only 2 files have to be changed

**Comment by andresilveirah** [comment]
@carmelo-iriti what's your suggestion?

**Review by carmelo-iriti — CHANGES_REQUESTED** [review]

**Comment by carmelo-iriti** [comment]
@andresilveirah my comment was pending 🤦🏼

**Comment by carmelo-iriti** [comment]
Sorry guys 😅

File: cmplibrary/src/release/java/com.sourcepoint.cmplibrary.exception/LoggerImpl.kt [hunk 1]
```diff
@@ -55,7 +57,12 @@ private class LoggerImpl(
     override fun error(e: RuntimeException) {
         val mediaType = "application/json".toMediaTypeOrNull()
         val body: RequestBody = RequestBody.create(mediaType, errorMessageManager.build(e))
-        val request: Request = Request.Builder().url(url).post(body)
+
```
Comment (carmelo-iriti) [inline cmplibrary/src/release/java/com.sourcepoint.cmplibrary.exception/LoggerImpl.kt:60]: This change is perfect 👍

Comment (Nevazhnovu) [reply cmplibrary/src/release/java/com.sourcepoint.cmplibrary.exception/LoggerImpl.kt:60]: Thanks!

**Review by carmelo-iriti — APPROVED** [review]

**Pull request merged**